stdlib/bytes.ml

```diff
@@ -97,19 +97,30 @@ let iter f a =
 let iteri f a =
   for i = 0 to length a - 1 do f i (unsafe_get a i) done
 
+let () = assert (3 * Sys.max_string_length <= max_int)
```

Comment:
Do we really want to run this in every ocaml program ? Besides the assertion will fail in bucklescript since they plan to move to `Sys.max_string_length = max_int`